genesis_core/user_api/iam/dm/models.py

```diff
@@ -1037,19 +1037,19 @@ def get_token_by_password(self, username, **kwargs):
         Get auth token by username + password (default approach).
         """
         users_query = User.objects.get_all(
-            filters={"name": ra_filters.EQ(username)}
+            filters={"LOWER(name)": ra_filters.EQ(username.lower())}
```

Comment:
Didn't know that "LOWER(name)" work as a key :) Hope it's not a bug.

Reply:
It doesnt, unfortunately. I'm still working on it.

Reply:
Got it, I've converted this PR in draft then.